Backend/databaseInteraction/getUserById.js

```diff
@@ -0,0 +1,20 @@
+const {models} = require('../sequelizeSetup/sequalizeConstructor');
+
+/**
+ * Gets User by Id
+ * 
+ * @param {int} id id of user in db
+ * @returns {Promise<object?>} a promise evaluating to the user object, if found, otherwise null
+ */
+function getUserById(id) {
```

Comment:
If you have a chance, can you write unit tests for `getUserById` in the `databaseInterface.test.js` file? We have them for all the other methods that pull from the database to verify that the output is as expected. Not necessary though